InternshipBE/BL/Interfaces/IInternshipService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using BL.DTOs;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace BL.Interfaces
+{
+    public interface IInternshipService
+    {
+        Task<InternshipDTO> GetInternshipAsync(int id);
+
+        Task<InternshipDTO> CreateInternshipAsync(InternshipDTO newInternship);
+
+        Task<InternshipDTO> UpdateInternshipAsync(InternshipDTO newInternship);
+
+        Task<List<InternshipDTO>> GetInternshipsAsync(int itemsCount, int pageNumber);
```

Comment:
pageSize

Reply:
Ahaha, how do you do it? 😅
I thought I fixed it everywhere 🙃